tensorflow/lite/tools/optimize/modify_model_interface.cc

```diff
@@ -46,7 +46,8 @@ struct TensorOpTensor {
 // Finds float tensors that are model inputs and is consumed by a quantize Op.
 // The returned TensorOpTensor should have reverse order.
 std::vector<TensorOpTensor> GetInputTensors(ModelT* model,
-                                            ErrorReporter* error_reporter) {
+                                            ErrorReporter* error_reporter,
+                                            const TensorType& input_type) {
```

Comment:
nit: prefer move immutable arguments before mutable arguments

Reply:
done, thank you!